udp-rsc-offload.md

```diff
@@ -27,6 +27,9 @@ This section describes necessary updates in the Windows network stack to support
 - If hardware URO is not available, but a socket opts-in to URO with the Winsock API, software URO will be used for that socket. Software coalescing will follow the same rules as NDIS, except for the following:
     - Software coalescing may decide to use multiple NBLs in a chain of up to 255, instead of a single NBL.
```

Comment:
This strikes me as both an implementation detail and a somewhat contradictory statement: if URO is grouping multiple NBLs into one NBL, then what is software URO doing if it takes multiple NBLs and returns multiple NBLs? One can imagine how this statement is, say, returning half as many NBLs, but it is still not a useful contract to document IMO.